Formula/nfdump.rb

```diff
@@ -11,8 +11,16 @@ class Nfdump < Formula
     sha256 "5bbd8ec9a51cde3c6ed0aaeb1b77a4f2ba59bea99d5304fa1415a7e2e0d5280b" => :mountain_lion
   end
 
+  option "with-readpcap", "Read netflow packets from a give pcap_file instead of the network. Is intended for debugging only."
```

Comment:
Okay, I see it in `man nfcapd`.

I think enabling it by default still makes sense. OS X ships with libpcap, so it's not even causing users to take another dependency. Even if it's a debugging tool, it doesn't sound like it's a dangerous thing we need to protect users from. Sounds more like it's optional just to avoid the libpcap dependency for people who wouldn't need that debugging tool.